apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +136,73 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor's name of the WebSoc section",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "Selects WebSoc sections held on any of the given days",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building of the WebSoc section",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room of the WebSoc section",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description: "The division level of the WebSoc section",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "The section type of the WebSoc section, see [Registrar](https://reg.uci.edu/)",
```

Comment:
If you link a page at all, it should be the exact page which describes the meaning of these codes. Linking the registrar's homepage is not helpful.